presto-main/src/main/java/com/facebook/presto/operator/aggregation/GenericAccumulatorFactory.java

```diff
@@ -151,10 +166,33 @@ public Accumulator createIntermediateAccumulator()
 
     @Override
     public GroupedAccumulator createGroupedAccumulator()
+    {
+        return spillEnabled ? createSpillableGroupedAccumulator() : createDefaultGroupedAccumulator();
+    }
+
+    @Override
+    public GroupedAccumulator createGroupedIntermediateAccumulator()
+    {
+        return spillEnabled ? createSpillableGroupedIntermediateAccumulator() : createDefaultGroupedAccumulator();
```

Comment:
For `spillEnabled  == false`, both returns `createDefaultGroupedAccumulator()`? That doesn't look right? Do we have unit tests to verify this?

Reply:
This is a copy paste bug from my last change.